reporting/api/src/main/java/org/jboss/windup/reporting/model/ClassificationModel.java

```diff
@@ -97,4 +104,12 @@
     @SetInProperties(propertyPrefix = "tag")
     Set<String> getTags();
 
+    abstract class Impl implements ClassificationModel, JavaHandlerContext<Vertex>
```

Comment:
I wanted to make FileModelTransformable as a needed interface.. It is because multiple Models may end up extending the interface (Hint, JavaTypeReferenceModel etc.)